### DISCUSSION TITLE
No such module 'Supabase'

### DISCUSSION THREAD

**Discussion by Iskhak04** [supabase/supabase#19764] [Questions]
I'm getting this error: No such module 'Supabase', even though I added Supabase-swift through Package Manager. Tried cleaning the build folder, deleting derived data, removing and adding Supabase-swift, deleting and installing Xcode again, nothing worked. A few days ago though everything was working fine on another project. I would really appreciate any insight.

**Comment by mabuex** [comment]
I had the same issue, not sure if it will help you but try  Project -> TARGETS("Project name") Frameworks, Libraries, and Embedded Content. 
Check if there is "Supabase" under Name if not try to add it by clicking the plus icon.
<img width="1071" alt="Screenshot 2023-12-23 at 3 17 23 PM" src="https://github.com/supabase/supabase/assets/103418376/29ed2503-3883-4788-9d3a-a01d60ac2629">

**Reply by mlawlerau** [reply]
Thanks Mabuex! That worked for me.

**Reply by paneiOS** [reply]
Thank you. I was able to resolve the issue thanks to your help.

**Reply by zhipengbird** [reply]
Thank you.

**Reply by Azure-Defiant** [reply]
thank you fellow developer

**Reply by zcsabbagh** [reply]
Thank you so much! This worked for me too

**Reply by rachellarralde** [reply]
Thank you!

**Reply by spicypunk** [reply]
thank you

**Reply by danielwanja** [reply]
That did the trick. Thanks!

**Reply by piavgh** [reply]
thank you, king

**Reply by Ogabek-Bakhodirov** [reply]
Thank you

**Reply by JAE0Y2N** [reply]
Thank you!

**Reply by nernoel** [reply]
Thank you!

**Reply by SilisK** [reply]
That worked, thanks!

**Comment by chrisdoyleIE** [comment]
Can say with 100% certainty that this will trump a lot of vibe coders, just dug myself out of an unecessarily complicated dependency hell hole. Please fix this supabase 🙏

**Reply by sonjay97** [reply]
It did indeed trump me as a vibe coder. I'm a python back-end guy. Never worked with Swift so it's a bit new navigating Xcode.

**Comment by sonjay97** [comment]
wanted to provide a screenshot to show other fellow vibe coders who might still be confused even after looking at the original answer.

<img width="1141" alt="xcode guide" src="https://github.com/user-attachments/assets/968bc00c-6051-4891-be9c-0767b9e0cd3b" />

**Comment by archisvaze** [comment]
Thanks!
